spec/models/drip_email_list_spec.rb

```diff
@@ -0,0 +1,19 @@
+# License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
+require 'rails_helper'
+
+RSpec.describe DripEmailList, type: :model do
+
+  describe '#list_path' do
+    it 'is correctly generated' do
+      email_list = build_stubbed(:email_list_base)
```

Comment:
This would need to be `build_stubbed(:drip_email_list_base)`. We're only testing how the DripEmailList class works here and that's one of the factories we're using.